src/main/java/VisualLogic/variables/VS1DString.java

```diff
@@ -78,30 +81,32 @@ public void copyValueFrom(Object in) {
     }
 
     public void loadFromStream(java.io.FileInputStream fis) {
-        try {
-            DataInputStream dis = new DataInputStream(fis);
+        try (DataInputStream dis = new DataInputStream(fis)) {
+
 
             int size = dis.readInt();
             value = new String[size];
 
             for (int i = 0; i < value.length; i++) {
                 value[i] = dis.readUTF();
             }
-        } catch (Exception ex) {
+        } catch (IOException ex) {
+            Logger.error(ex);
             System.err.println("Fehler in VS1DString.loadFromStream() : " + ex.toString());
```

Comment:
`System.err` call should be removed. It might be worth while to search for all uses of it. Won't comment on other occurrences.